addons/web/static/src/webclient/navbar/navbar.scss

```diff
@@ -150,8 +149,12 @@
         border-color: transparent;
     }
 
-    .o_menu_sections {
-        .o_nav_entry, .dropdown-toggle {
+    .o_menu_sections, .o_menu_systray:where(:not(.o_home_menu_background .o_menu_systray)) {
+        .dropdown-toggle:disabled {
+            --NavBar-entry-color--hover: var(--NavBar-entry-color, #{$o-navbar-entry-color});
+        }
+
+        .o_nav_entry, .dropdown-toggle:not([disabled]) {
```

Comment:
Is it a reason why you use `:disabled` in one selector and `[disabled]` in the the other ? 🤔